packages/clay-shared/src/getEllipsisItems.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import React from 'react';
+
+const getBufferList = (
```

Comment:
Instead of having a ton of arguments, lets just make this accept a config object. Usually is there are more than 2 or 3 args, its easier to pass in an object since its more explicit.

Reply:
Done!